frigate/ffmpeg_presets.py

```diff
@@ -114,6 +114,21 @@ def get_selected_gpu(self) -> str:
     "default": "ffmpeg -hide_banner {0} -c:v libx264 -preset:v ultrafast -tune:v zerolatency {1}",
 }
 
+PRESETS_HW_ACCEL_ENCODE_PREVIEW = {
+    "preset-rpi-64-h264": "ffmpeg -hide_banner {0} -c:v h264_v4l2m2m -pix_fmt yuv420p {1}",
```

Comment:
The profile can make a big difference in CPU usage here. May be good to look at ultrafast for all of these.

Reply:
makes sense to use base profile for all, I don't think hardware encoders support libx264 presets like ultrafast

https://trac.ffmpeg.org/wiki/Hardware/VAAPI#Mappingoptionsfromlibx264